sql/moz-fx-data-shared-prod/org_mozilla_fennec_aurora_derived/baseline_clients_daily_v1/backfill.yaml

```diff
@@ -0,0 +1,8 @@
+2024-05-22:
```

Comment:
@ANich does this need to be a date in the future? As in, is this used by our job to determine which config to pick up? If yes, would this need to be changed to tomorrow's date so that next days run catches it?